src/Lecture_4/Homework_4/Task_11.java

```diff
@@ -0,0 +1,25 @@
+package Lecture_4.Homework_4;
+
+import java.util.Scanner;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class Task_11 {
+
+    public static void main(String[] args) {
+
+        Scanner sn = new Scanner(System.in);
+
+        System.out.println("Please enter an email");
+        String input = sn.nextLine();
+
+        Pattern pattern = Pattern.compile("^[A-Za-z0-9+_.-]+@(.+)$");
```

Comment:
Based on your regex this are valid emails:
bl....a@abv.bggggggg
bl....a@abv
bl@@@@@a@abv.

Reply:
The last one looks especially tricky.